script/update_example_glean_usage

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+## Updates example queries that are checked into the default branch for testing.
+
+set -e
+
+cd "$(dirname "$0")/.."
+
+scripts=(
+    "run_glean_baseline_clients_daily"
+    "run_glean_baseline_clients_first_seen"
+    "run_glean_baseline_clients_last_seen"
+)
+
+# kill all children in the case when we kill the parent script
+# https://stackoverflow.com/a/360275
+trap "jobs -p | xargs -r kill" EXIT
```

Comment:
Is this slow enough that running these concurrently is necessary?

Reply:
It took about 10 seconds at some point before I parallelized it even further, so it might be a bit overkill. It's also a handy snippet, so it might be worth keeping around .